packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -351,15 +357,31 @@ export class IosSimulatorDevice extends DeviceBase {
 
     await this.terminateAnyRunningApplications();
 
-    await exec("xcrun", [
+    if (this.runningAppProcess) {
+      this.runningAppProcess.kill(9);
```

Comment:
Just ensuring the process is killed - the weird logs won't come up after the new lunch starts. Thus, I can reuse the channel - recreating it requires the user to select output each time, which is lame.